packages/p2p/src/stores/sendbird-store.js

```diff
@@ -19,6 +19,8 @@ export default class SendbirdStore extends BaseStore {
     sendbird_api = null;
     service_token_timeout = null;
     scroll_debounce = null;
+    file_upload_properties = null;
+    is_upload_complete = false;
```

Comment:
***Nit*** Make sure to put all the variables in alphabetical order, same with when adding new functions, just to keep things clean